bitnami/sealed-secrets/templates/role.yaml

```
@@ -57,3 +64,61 @@ rules:
   {{- include "common.tplvalues.render" ( dict "value" .Values.rbac.serviceProxier.rules "context" $ ) | nindent 2 }}
   {{- end }}
 {{- end }}
+{{ if and (and .Values.rbac.create .Values.rbac.namespacedRoles) (not $.Values.rbac.clusterRole) }}
```

```suggestion
{{ if and (and .Values.rbac.create .Values.rbac.namespacedRoles) (not .Values.rbac.clusterRole) }}
```
Same suggestion for the `clusterrole.yaml` template.